pages/concepts/bank-funding-source.mdx

```diff
@@ -0,0 +1,149 @@
+---
+layout: concepts
+title: "Bank Funding Source"
+description: Send or receive funds to or from a bank funding source for client master accounts and Customer accounts.
+category: Funding Sources
+concept:
+  icon: "balance-funding-source.svg"
```

Comment:
Requesting a new icon from Design. Meanwhile, we can use the same icon as the balance-funding-source.